hashdb.py

```diff
@@ -1152,12 +1165,15 @@ def set_xor_key():
     """
     global HASHDB_USE_XOR
     global HASHDB_XOR_VALUE
-    xor_value = parse_highlighted_value("ERROR: Not a valid xor selection\n")
+    xor_value = parse_highlighted_value()
     if xor_value is None:
+        idaapi.msg("HashDB ERROR: Invalid xor value selection.\n")
```

Comment:
Thanks for fixing this, I never liked the error msg passing... sub-functionality should only return status, leave printing the status up to the caller